vmdb/app/models/dialog_field_text_box.rb

```diff
@@ -42,6 +42,8 @@ def automate_key_name
   end
 
   def validate(dialog_tab, dialog_group)
+    return if !required? && value.blank?
```

Comment:
Should it return a literal false or something else?  This just returns nil, which although it's falsy, makes it appear more like the code is saying "meh, I don't care about this"